core/src/main/java/feign/Logger.java

```diff
@@ -107,7 +107,7 @@ protected Response logAndRebufferResponse(String configKey, Level logLevel, Resp
     return response;
   }
 
-  protected IOException logIOException(String configKey, Level logLevel, IOException ioe, long elapsedTime) {
+  protected void logIOException(String configKey, Level logLevel, IOException ioe, long elapsedTime) {
```

Comment:
1. The return value is unused. 
1. The method is neither a command nor a query.